wsd/COOLWSD.cpp

```diff
@@ -5056,6 +5059,7 @@ int COOLWSD::innerMain()
 #endif
 
     const auto startStamp = std::chrono::steady_clock::now();
+    auto stampFetch = startStamp;
```

Comment:
Lets subtract the durationFetch from that and add say 1 minute - so we fetch early after startup the first time.

Reply:
done